package.json

```diff
@@ -45,7 +45,9 @@
       "manifesto.js": "^4.2.17",
       "mariadb": "^3.2.3",
       "mongodb": "^6.4.0",
-      "morgan": "^1.10.0"
+      "morgan": "^1.10.0",
+      "nodemailer": "^6.9.14",
+      "tpen3-services": "file:"
```

Comment:
`"tpen3-services": "file:"` is this supposed to be here?

Reply:
@cubap